workers/unity/Packages/com.improbable.gdk.querybasedinteresthelper/InterestHelper.cs

```diff
@@ -0,0 +1,220 @@
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.Gdk.Core;
+
+namespace Improbable.Gdk.QueryBasedInterest
+{
+    /// <summary>
+    ///     Utility class to help construct Interest component snapshots.
+    /// </summary>
+    public class InterestHelper
```

Comment:
This name could be clearer. It seems like it is a way of creating a dictionary of specific queries. I generally wouldn't expect to instantiate something called InterestHelper and save them as archetypal queries. I would probably name that a noun.